### DISCUSSION TITLE
Are there any plans for s390x/mongo images with newer Mongo versions?

### DISCUSSION THREAD

**Discussion by paulben** [docker-library/mongo#572] [General]
Are there any plans for s390x/mongo images with newer Mongo versions?

Can anyone tell me who maintains the s390x/mongo images @ https://hub.docker.com/r/s390x/mongo ?

Thanks, Paul Bennett

**Comment by yosifkit** [comment]
We own `s390x/mongo` (and when it was active, they would be part of the manifest list that makes up https://hub.docker.com/_/mongo/). MongoDB stopped providing updates to the community version on Ubuntu (18.04) for s390x and doesn't have s390x for any newer version of Ubuntu  https://www.mongodb.com/docs/manual/administration/production-notes/#platform-support-matrix.

Commit that dropped s390x on 4.4: https://github.com/docker-library/mongo/commit/babc077da86d7ab12a225300feb71fa3b309b1da. And on 4.2: https://github.com/docker-library/mongo/commit/cd315c60d63f034cf9556dc947b90a92eb535f09
